src/main/kotlin/com/slackuserpresencemanager/Main.kt

```diff
@@ -0,0 +1,41 @@
+package com.slackuserpresencemanager
+
+import com.slackuserpresencemanager.listeners.WindowsSessionManager
+import com.slackuserpresencemanager.listeners.WindowsStateManager
+import org.apache.logging.log4j.LogManager
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.util.Properties
+
+/**
+ * Created by Tharaka on 4/15/2017.
+ * Project: SlackUserPresenceManager
+ */
+object Main {
+
+    private val LOGGER = LogManager.getLogger(Main::class.java)
+
+    private val PROPERTY_FILE_NAME = "SlackUserPresenceManager.properties"
+
+    private val properties = Properties()
+
+    fun getProperty(key: String): String {
+        return properties.getProperty(key, "")
+    }
+
+    init {
```

Comment:
@tharakadesilva is init just the main method for Kotlin?

Reply:
init is the contstructor